S2VX.Game.Tests/CommandTests.cs

```diff
@@ -0,0 +1,150 @@
+﻿using NUnit.Framework;
+using osu.Framework.Graphics;
+using osu.Framework.Utils;
+using osuTK;
+using osuTK.Graphics;
+using S2VX.Game.Story.Command;
+using System;
+
+namespace S2VX.Game.Tests {
+    [TestFixture]
+    public class CommandTests {
+        [Test]
+        public void FromString_DoubleCommand() {
+            var input = "ApproachesDistance|0.1|0.2|None|0.1|1.0";
+            var expected = new ApproachesDistanceCommand() {
+                StartTime = 0.1f,
+                EndTime = 0.2f,
+                Easing = Enum.Parse<Easing>("None"),
+                StartValue = 0.1f,
+                EndValue = 1.0f,
+            };
+            var actual = (ApproachesDistanceCommand)S2VXCommand.FromString(input);
+            var tolerance = 0.001;
```

Comment:
I think this can be extracted out to a reusable static variable

Reply:
I'll make it a const